### DISCUSSION TITLE
Wie benutzt Ihr die Listen dieses Projekts?

### DISCUSSION THREAD

**Discussion by RPiList** [RPiList/specials#1090] [General]
Hi,

aufgrund der neuen ABP-Style Funktion in Pihole werden nach und nach entsprechende Subdomains aus den Listen entfernt. D.h. aus:

de.pornsite.com
uk.pornsite.com
m.pornsite.com
www.pornsite.com
pornsite.com

in der alten Liste wird

||pornsite.com^

in der neuen Liste.

Wird die Liste in Programmen genutzt, die mit dem sog. ABP-Style umgehen können, wird der entsprechende Schutz für die Zielgruppe erhöht. 

Meine Frage ist aber ob die Liste evtl. noch irgendwo zum Einsatz kommt, in der die neue Formatierung nicht funktioniert. Weil da geht der Schutz dann evtl. flöten. In dem Fall, wenn die Community das wünscht, könnte man drüber nachdenken die "alten" Listen auch weiterzuführen.

Daher bitte mal Feedback liefern.

**Comment by yoshimo** [comment]
Wie alle andren Listen aus dem Netz auch werden sie sowohl im Pi-Hole genutzt als auch auf den jeweiligen Endgeräten selbst.
Hosts-Format macht es einfach bequem um mit AdAway oder direkt auf nem DesktopPC benutzt zu werden.

**Comment by Bloodpack** [comment]
Ich nutze das "alte" format mit pfsense/pfblocker

**Comment by chriss199815** [comment]
Ich komme gerade aus dem Stream. Und habe erfahren das man das Alte Format auch unter PfSense nutzen kann da ich gerade dabei bin so etwas aufzubauen, würde ich mich freuen wenn das alte Format bei behalten wird.

**Comment by Copy05** [comment]
Diese liste hat mir paar subdomäne gezeigt die wirklich interesant sind und die ich in meinem Subdomain Enumeration eingeflügt hab: [Hconsole [src/hsubenum.c]](https://github.com/Copy05/Hconsole/blob/main/src/hcsubenum.c)
